src/presentation/components/modal-select-network.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useRef } from 'react';
+import useOnClickOutside from '../hooks/use-onclick-outside';
+
+interface Props {
+  isOpen: boolean;
+  onClose: () => any;
+  onLightning: any;
+  onLiquid: any;
+}
+
+const ModalSelectNetwork: React.FC<Props> = ({ isOpen, onClose, onLightning, onLiquid }) => {
+  const ref = useRef<HTMLDivElement | null>(null);
+  useOnClickOutside(ref, onClose);
+
+  if (!isOpen) return <></>;
+
+  return (
+    <div className="fixed bottom-0 z-50 flex">
```

Comment:
some shadow to be consistent with Figma?

Reply:
Fixed by eae4e0d